Teams_App/src/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
-<body>
+<body style="margin: 0px;">
     <div id="root"></div>
     <script src="./App.js" type="module"></script>
```

Comment:
Es6 modules are deferred by default. You can use them in head tag.

Reply:
Does adding to the body have any downside ? I generally keep it at the bottom of body because it is more natural to read because we are used to seeing script tags at the bottom